ui/app/AppLayouts/Communities/views/MintedTokensView.qml

```diff
@@ -52,12 +76,28 @@ StatusScrollView {
         width: root.viewWidth
         spacing: Style.current.halfPadding
 
+        IntroPanel {
+            Layout.fillWidth: true
+
+            visible: root.count === 0
+
+            image: Style.png("community/mint2_1")
+            title: qsTr("Community tokens")
+            subtitle: qsTr("You can mint custom tokens and import tokens for your community")
+            checkersModel: [
+                qsTr("Create remotely destructible soulbound tokens for admin permissions"),
+                qsTr("Reward individual members with custom tokens for their contribution"),
+                qsTr("Mint tokens for use with community and channel permissions")
+            ]
+        }
+
         StatusBaseText {
             Layout.leftMargin: Style.current.padding
 
             text: qsTr("Assets")
             font.pixelSize: Theme.primaryTextFontSize
             color: Theme.palette.baseColor1
+            visible: root.count !== 0
```

Comment:
Instead of having this checker in each component below, I guess we can have a component that includes all of them and just show or hide the component on this condition (or as said before, with a loader and change source component).

Reply:
Done